jquery.liteuploader.js

```diff
@@ -22,7 +28,7 @@ function LiteUploader (element, options) {
     this.el = $(element);
     this.options = options;
     this.params = options.params;
-    this.xhr = this._buildXhrObject();
+    this.xhrs = [];
```

Comment:
This holds all the concurrent XHR Objects, to be able to abort them all by the client, there's no other purpose for this.
The first object is created on the first upload, not before.